puara.cpp

```diff
@@ -1497,17 +1501,17 @@ std::string Puara::getLocalPORTStr() {
 }
 
 bool Puara::IP1_ready() {
-    if (oscIP1 != "0.0.0.0" || oscIP1 != "") {
+    if ((oscIP1 != "0.0.0.0") && (oscIP1 != "")) {
         return true;
     } else {
         return false;
     }
 }
```

Comment:
Another possibility to make it more readable IMO is 

```suggestion
    if ((oscIP1 == "0.0.0.0") || (oscIP1 == "")) {
        return false;
    } else {
        return true;
    }
}
```

We can also use the ternary operator (I think makes it more readable in this case):

`return (oscIP1 == "0.0.0.0" || oscIP1 == "") ? false : true;`

Reply:
definitely, I'll make the change